### PR TITLE
types: Allow []byte as orderby type

### PR DESCRIPTION
**Pull request by shenli**
Orderby []byte is already supported by types.Compare.
With this pr we can run most of functions in phpMyAdmin.

### CODE REVIEW

**Comment by qiuyesuifeng** [comment]
LGTM

File: util/types/etc.go [hunk 1]
```diff
@@ -298,7 +298,7 @@ func IsOrderedType(v interface{}) (y interface{}, r bool, err error) {
 	case float32, float64,
 		int, int8, int16, int32, int64,
 		uint, uint8, uint16, uint32, uint64,
-		string, mysql.Decimal:
+		string, mysql.Decimal, []byte:
 		return v, true, nil
 	case mysql.Time, mysql.Duration:
```
Comment (siddontang) [inline util/types/etc.go]: can move following Time and Duration here too.

Comment (shenli) [inline util/types/etc.go]: Can we remove the first return value?

Comment (siddontang) [inline util/types/etc.go]: I think we can just return true/false too.

**Comment by shenli** [comment]
PTAL @siddontang

**Pull request merged**